### PR TITLE
hardening/640_add_hw_requirements

### PR DESCRIPTION
**Pull request by pcoello25**
* Implements/Fixes issue #640 
* No tests are required
* Assignee @frbattid

### CODE REVIEW

File: doc/installation_and_administration_guide/hw_requirements.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* RAM: 1 GB, specially if abusing of the batching mechanism.
```
Comment (frbattid) [inline doc/installation_and_administration_guide/hw_requirements.md]: Some header/title is required here, e.g. "Hardware requirements"

File: README.md [hunk 2]
```diff
@@ -37,7 +38,13 @@ You may consider to visit [Cygnus Quick Start Guide](doc/quick_start_guide.md) b
 [Top](#top)
 
 ##<a name="section2"></a>Basic operation
-###<a name="section2.1"></a>Installation (CentOS/RedHat)
+###<a name="section2.1"></a>Hardware requirements 
+*  RAM: 1 GB, specially if abusing of the batching mechanism.
```
Comment (frbattid) [inline README.md]: It seems there are two spaces between `*` and the text.

File: README.md [hunk 3]
```diff
@@ -37,7 +38,13 @@ You may consider to visit [Cygnus Quick Start Guide](doc/quick_start_guide.md) b
...
+*  HDD: A few GB may be enough unless the channel types are configured as FileChannel type.
```
Comment (frbattid) [inline README.md]: FileChannel must be written as `FileChannel`. I mean, as code text, using ` and ` (edit this comment in order to see how to do that ;)).

File: doc/installation_and_administration_guide/hw_requirements.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+* HDD: A few GB may be enough unless the channel types are configured as FileChannel type.
```
Comment (frbattid) [inline doc/installation_and_administration_guide/hw_requirements.md]: Same comment than above regarding `FileChannel` font.

**Comment by frbattid** [comment]
In addition to the comments on the files: the README within the installation and adminsitration guide must be edited as well, adding a subsection for the HW requirements.

File: doc/installation_and_administration_guide/README.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 #Installation and Administration Guide
 
 * [Introduction](./introduction.md)
+* [Hardware requirements](./hw_requirements.md)
```
Comment (frbattid) [inline doc/installation_and_administration_guide/README.md]: This must be a subsection within the installation section.

File: doc/installation_and_administration_guide/hw_requirements.md (hunk 1 above)
Comment (pcoello25) [inline doc/installation_and_administration_guide/hw_requirements.md]: Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>

File: README.md (hunk 2 above)
Comment (pcoello25) [inline README.md]: Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>

File: README.md (hunk 3 above)
Comment (pcoello25) [inline README.md]: Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>

File: doc/installation_and_administration_guide/hw_requirements.md (hunk 4 above)
Comment (pcoello25) [inline doc/installation_and_administration_guide/hw_requirements.md]: Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>

File: doc/installation_and_administration_guide/README.md (hunk 5 above)
Comment (pcoello25) [inline doc/installation_and_administration_guide/README.md]: Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>

**Comment by frbattid** [comment]
LGTM. Congratulations for your first PR, @pcoello25 !!

**Pull request merged**